Update the documentation (#15)
fastlane/github-actions: README.md

> ## GitHub Actions Maintainer 

Reviewer: What does "Maintainer" mean here?

Author: I asked myself a question "what this repository is all about"? To me it is a set of GitHub Actions that help maintainers to maintain `fastlane` project.  That's why I called it `GitHub Actions Maintainer`. Any suggestions how to improve this name @janpio ?

Reviewer: "Fastlane GitHub Actions" ;)

Reviewer: "A set of GitHub Actions fastlane uses."

> The GitHub Actions Maintainer provides a set of GitHub Actions to make maitaining project easier.

Reviewer: ```suggestion
The GitHub Actions Maintainer provides a set of GitHub Actions to make maintaining projects easier.
```
What projects?

Author: The projects that will use the set of actions from this repository.

## Actions

> 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder) 

Reviewer: ```suggestion
- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder) 
```

> Adds a comment to include fastlane environment in an issue description if it is missing. Read more [here](fastlane-env-reminder).

Reviewer: ```suggestion
   Adds a comment to include fastlane environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).
```

## Versioning 

> All the actions are released in one batch. We do not support semantic versioning. Reference a `latest` branch in your workflow:

Reviewer: ```suggestion
All the actions are released in one batch. We do not support semantic versioning (yet). Reference a `latest` branch in your workflow:
```

```yaml
...
  - uses: fastlane/github-actions/fastlane-env-reminder@latest
...
```

## Contributing

> We welcome contributions.  See [how to contribute](CONTRIBUTING.md).

Reviewer: ```suggestion
We welcome contributions. See [how to contribute](CONTRIBUTING.md).
```

## GitHub Actions Maintainer 

The GitHub Actions Maintainer provides a set of GitHub Actions to make maintaining projects easier.

## Actions

- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)

>    Adds a comment to include fastlane environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).

Reviewer 2: ```suggestion
   Adds a comment to include _fastlane_ environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).
```

## Fastlane GitHub Actions

The Fastlane GitHub Actions provide a set of GitHub Actions to make maintaining projects easier.

## Actions

- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)

   Adds a comment to include _fastlane_ environment information in an issue description if it is missing. Read more [here](fastlane-env-reminder).
   

> - ✔️ [@github-actions/communicate-on-pull-request-merged](communicate-on-pull-request-merged)

Reviewer: This link does not work yet.

Author: That's correct. Do you think we should remove it from the docs then? 🤔 The PR is ready and I hope it will be merged pretty soon, so I'd suggest to leave it 🚀

Reviewer: Nah, then this is perfectly fine.

   Adds a comment and a label to a pull request when it is merged. Read more [here](communicate-on-pull-request-merged).

## Versioning 

All the actions are released in one batch. We do not support semantic versioning (yet). Reference a `latest` branch in your workflow:

```yaml
...

>   - uses: fastlane/github-actions/fastlane-env-reminder@latest

Reviewer: Maybe add the second action here somehow as well? COnfused me why this one was singled out.

Author: ![image](https://user-images.githubusercontent.com/10795657/68889184-b5468d80-071c-11ea-97bb-047880e636a3.png)

To put the usage in a context, this section is about how to use the action in a workflow. Do you have any ideas how we could improve the description to avoid the confusion you get? 🤔

Reviewer: Nope, not really - I think this is fine.

## Fastlane GitHub Actions

> The Fastlane GitHub Actions provides a set of GitHub Actions to make maintaining projects easier.

Reviewer: ```suggestion
The Fastlane GitHub Actions provide a set of GitHub Actions to make maintaining projects easier.
```

## Actions

- 🌏 [@github-actions/fastlane-env-reminder](fastlane-env-reminder)

> 

Reviewer: You can probably get rid of these two empty lines here (8 + 9) and below (13 + 14)

Author: I removed just one line to keep a description in a new line 👍
